### PR TITLE
QOL: see DaggerfallLoot items listed in an Inspector window (EditorOnly)

### PR DESCRIPTION
**Pull request by andrew-raphael-lukasik**
Hi,

`DaggerfallLootEditor` adds ability to see `DaggerfallLoot`'s items listed in the Inspect window. Could be useful for casual in-editor debugging.

It displays "`ItemName` x `stackCount`", read-only, at the moment.

j0reh32q4woer90gh34q 'rae0gvph;;4qgrjw6b

___

e0c5ba26791323465fe1dc2bde71e0ef24b6217d introduces item icons:

Screenshot 2022-04-24 123337

### CODE REVIEW

File: Assets/Scripts/Editor/DaggerfallLootEditor.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+using DaggerfallConnect.FallExe;
+using DaggerfallWorkshop.Game.Entity;
+
+namespace DaggerfallWorkshop
+{
+    [CustomEditor(typeof(DaggerfallLoot))]
+    [CanEditMultipleObjects]
+    public class DaggerfallLootEditor : Editor
+    {
+        DaggerfallLoot _target;
```
[the remark this answers is not in the document]
Comment (andrew-raphael-lukasik) [reply Assets/Scripts/Editor/DaggerfallLootEditor.cs:30]: Thank you. I'm on it

[the remark this answers is not in the document]
Comment (andrew-raphael-lukasik) [reply Assets/Scripts/Editor/DaggerfallLootEditor.cs]: a38150c451e4b4cd484cd5ed04442c0800cf04f0 fixes the issue

**Comment by andrew-raphael-lukasik** [comment]
e0c5ba26791323465fe1dc2bde71e0ef24b6217d introduces an item icon:

Screenshot 2022-04-24 123337

**Comment by Interkarma** [comment]
This is very cool, and works well. Thanks for the contributions. :)

**Pull request merged**